src/index.js

```diff
@@ -5,37 +5,68 @@
 import './css/base.scss';
 import fetcher from './fetch.js';
 import User from './User.js';
+import domUpdates from './DOM-updates.js';
+
+let selectionBox = document.querySelector('.selection-box');
 
 let tripsToDisplay, allDestinations, user;
 
 window.onload = fetchStuff;
-window.addEventListener('click', clickLog);
+selectionBox.addEventListener('click', clickLog);
 
 function fetchStuff() {
-    fetcher.fetchUser(7);
-    fetcher.fetchDestination();
+    let promisededUser = fetcher.fetchUser(7);
+    let promisededAllTrips = fetcher.fetchTripsForAUser();
+    let promisededAllDestinations = fetcher.fetchDestination();
+
+    Promise.all([promisededUser, promisededAllTrips, promisededAllDestinations])
```

Comment:
Nice implementation of `Promise.all()`! Looks great!

Reply:
![](https://media2.giphy.com/media/11kTQgng5gbqgM/giphy.gif)